joinservice/joinproto/join.proto

```diff
@@ -9,14 +9,17 @@ option go_package = "github.com/edgelesssys/constellation/v2/joinservice/joinpro
 service API {
   // IssueJoinTicket issues a join ticket for a new node.
   rpc IssueJoinTicket(IssueJoinTicketRequest) returns (IssueJoinTicketResponse);
-  // IssueRejoinTicket issues a join ticket for a node that has previously joined the cluster.
-  rpc IssueRejoinTicket(IssueRejoinTicketRequest) returns (IssueRejoinTicketResponse);
+  // IssueRejoinTicket issues a join ticket for a node that has previously
+  // joined the cluster.
+  rpc IssueRejoinTicket(IssueRejoinTicketRequest)
+      returns (IssueRejoinTicketResponse);
```

Comment:
Please don't reformat existing code if you're not actually changing it - that makes it harder to inspect the history.